delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/organizations/OrganizationsImpl.scala

```diff
@@ -138,16 +136,29 @@ object OrganizationsImpl {
       config: OrganizationsConfig,
       eventLog: EventLog[Envelope[OrganizationEvent]],
       index: OrganizationsCache,
-      orgs: Organizations
+      orgs: Organizations,
+      si: Set[ScopeInitialization]
   )(implicit as: ActorSystem[Nothing], sc: Scheduler) =
     StreamSupervisor(
       "OrganizationsIndex",
       streamTask = Task.delay(
         eventLog
           .eventsByTag(moduleType, Offset.noOffset)
-          .mapAsync(config.indexing.concurrency)(envelope =>
-            orgs.fetch(envelope.event.label).redeemCauseWith(_ => IO.unit, res => index.put(res.value.label, res))
-          )
+          .mapAsync(config.indexing.concurrency) { envelope =>
+            orgs
+              .fetch(envelope.event.label)
+              .redeemCauseWith(
+                _ => IO.unit,
+                { resource =>
+                  index.put(resource.value.label, resource) >>
+                    IO.when(!resource.deprecated) {
+                      IO.parTraverseUnordered(si)(_.onOrganizationCreation(resource.value, resource.createdBy))
+                        .void
+                        .redeemCause(_ => (), identity)
```

Comment:
We should have some logs somewhere to know when it fails and keep failing

Reply:
Each ScopeIntialization implementation records these logs.